src/main/target/common_pre.h

```diff
@@ -301,7 +301,7 @@
 #define USE_HUFFMAN
 
 #define PID_PROFILE_COUNT 4
-#define CONTROL_RATE_PROFILE_COUNT  4
+#define CONTROL_RATE_PROFILE_COUNT 6
```

Comment:
Configurator should handle any values, so use only default, but let user specify larger or smaller
```suggestion
#ifndef CONTROL_RATE_PROFILE_COUNT
#define CONTROL_RATE_PROFILE_COUNT 4 // or maybe 6
#endif
```

Reply:
If we want the number of rates profiles to be dynamic we need a MSP write message too (and not use MSP_STATUS_EX) as CONTROL_RATE_PROFILE_COUNT is used in several loops.

Reply:
Configurator just needs to know number of profiles available. MSP will refuse profiles that won't  fit and you can keep unused profiles at default values.
Number of profiles won't be dynamic, just build-time-configurable

Reply:
> Configurator should handle any values, so use only default, but let user specify larger or smaller

Ah see now you mean using CONTROL_RATE_PROFILE_COUNT as configurable define instead.

Reply:
yes, such was my initial thought as well. (in the issue-ticket)